SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,73 @@
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater or Equal than                   | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less than                               | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less or Equal than                      | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | XOR                                     | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Exponential                             | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Log                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Max                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Reciprocal                              | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Relu                                    | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Sigmoid                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Softmax                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Tanh                                    | - FSS        |                     2 |
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | Public           | Multiplication                          | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
```

Comment:
I think we  also have private multiplication for  spdz and fss

Reply:
Only SPDZ, not FSS.

Reply:
The FSS paper  implements beaver triples for multiplication(almost same operation), should we takes it as same as same or different.

Reply:
I checked the code and if I am not wrong, @kamathhrishi is correct. The beaver thing, I let it up for you to discuss.

Reply:
Yeah we could mention only SPDZ :+1:

Reply:
Perfect, I think I already corrected this. Whenever you have time check it. I left only the row for "Mult, matmul, conv2d, conv2d_transpose" with Falcon and SPDZ. That way I think it should be covered.